apps/dashboard/src/app/team/[team_slug]/[project_slug]/insight/components/BlueprintsExplorer.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import {} from "@/components/ui/dropdown-menu";
+import {} from "lucide-react";
+import Link from "next/link";
+
+import { ProjectAvatar } from "@/components/blocks/Avatars/ProjectAvatar";
+import {} from "@/components/ui/select";
+import { BillingAlerts } from "components/settings/Account/Billing/alerts/Alert";
+
+export type Blueprint = {
+  id: string;
+  name: string;
+  slug: string;
+  description: string;
+};
+
+export function BlueprintsExplorer(props: {
+  blueprints: Blueprint[];
+}) {
+  const { blueprints } = props;
+  return (
+    <div className="container px-0">
+      <BillingAlerts className="mb-10" />
+
+      {/* Blueprints */}
+      {blueprints.length === 0 ? (
+        <div className="flex h-[450px] items-center justify-center rounded-lg border border-border ">
+          No blueprints found
+        </div>
+      ) : (
+        <div className="grid grid-cols-1 gap-5 md:grid-cols-2 lg:grid-cols-3">
+          {blueprints.map((blueprint) => {
+            return <BlueprintCard key={blueprint.id} blueprint={blueprint} />;
+          })}
+        </div>
+      )}
+
+      <div className="h-10" />
+    </div>
+  );
+}
+
+function BlueprintCard(props: {
+  blueprint: Blueprint;
+}) {
+  const { blueprint } = props;
+  return (
+    <div
+      key={blueprint.id}
+      className="relative flex items-center gap-4 rounded-lg border border-border bg-muted/50 p-4 transition-colors hover:bg-muted/70"
+    >
+      <ProjectAvatar className="size-10 rounded-full" src="" />
```

Comment:
Don't use `ProjectAvatar` here - this is not a project link. use a different icon

Reply:
done